.github/workflows/order-id-validator.yml

```diff
@@ -0,0 +1,19 @@
+name: Order ID validation
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  validate-order-id:
+    runs-on: ubuntu-latest
+    name: Label issue based on order ID
+    steps:
+      - name: Label issue
+        uses: michelengelen/mui-order-id-validation@8ab4e02a38c644709820f4f4af1f8d8cd22a441e
```

Comment:
Why do we have these actions in an external repository instead of here? I've seen them in multiple places 😅

Reply:
Maybe it's easier that way? Or not pollute the org too much 🤷

Reply:
But we can have them in this repo if they are only needed here. If they are shared we could have a single repository where shared actions live.

To create the action in this repo you can create a `./.github/actions/my-action-name/action.yaml`

```yaml
name: My Action
description: Used for action

inputs:
  something:
    description: Is something
    required: true

runs:
  using: "composite"
  steps:
    - run: node ./my-script.js
```

Reply:
Also this is a very simple use-case that doesn't need to be repeated, so it could simply be just a workflow that runs node 😅 instead of having a whole action built for it.

Reply:
Those are superb points! 👍 
I also agree that having better visibility on those actions would be nice.
Especially this one, when it's only targeted at MUI X. 👍